test/controller.test.ts

```diff
@@ -108,11 +108,24 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
   })
 
   describe('Check if price is finalized', () => {
+    let expiredOtoken: MockOtokenInstance
+
     before(async () => {
       const lockingPeriod = new BigNumber(60) // 1min
       const disputePeriod = new BigNumber(60) // 1min
 
-      const batch = (await controller.getBatchDetails(otoken.address))[0]
+      expiredOtoken = await MockOtoken.new()
+      // init otoken
+      await expiredOtoken.init(
+        weth.address,
+        usdc.address,
+        usdc.address,
+        new BigNumber(200).times(new BigNumber(10).exponentiatedBy(18)),
+        new BigNumber(await time.latest()),
+        true,
+      )
+
+      const batch = (await controller.getBatchDetails(expiredOtoken.address))[0]
       // set batch oracle
       await oracle.setBatchOracle(batch, batchOracle.address)
```

Comment:
maybe you shouldn't be able to setBatchOraclePrice if the locking and dispute period is set in the first place?

Reply:
isn't that weird if you can set the price because then it automatically becomes finalized?

Reply:
mm what do u mean ?

Reply:
oh whoops I read it as price. nvm. all good.